src/Containers/NutritionFact/NutritionFact.tsx

```diff
@@ -0,0 +1,296 @@
+import React,{useCallback,useMemo,useRef,forwardRef,useEffect,useState} from 'react'
+import styled from 'styled-components'
+import { Mixins } from '../../Utils'
+import {
+    Main,
+    MainInterface,
+} from '../../Utils/BaseStyles';
+import Theme from '../../Themes/ThemeTemplate'
+import {Input} from '../../Inputs/Input/Input'
+
+const EXTRA_PIXEL=1
+const ENTRY_INPUT_WIDTH=20
+const HEADER_ENTRY_INPUT_WIDTH=40
+
+interface IAdditionalProps{
+    ref?:boolean;
+    delay?:boolean;
+}
+
+interface IEntries{
+    type:string;
+    entry:IEntryProps&IAdditionalProps|IHeadingEntryProps&IAdditionalProps;
```

Comment:
Try to use TypeScript Comments
Interface {
  /* Data Per Line */ <--- start writing this
  Entry: Type
}